lib/msal-browser/FAQ.md

```diff
@@ -82,12 +82,6 @@ MSAL.js has been tested and supports the last 2 stable and supported versions of
 - Safari
 - Opera
 
-MSAL.js has also been tested and supports the following browsers with Promise polyfills (not included):
```

Comment:
Should we add a note we are discontinuing support for these browsers?

Reply:
It'll be in the migration doc and release notes. That should be enough I think.

Reply:
There is a note at the end of v2 migration doc. It will be mentioned in the release notes as well.